cmd/ocm/create/cluster/cmd.go

```diff
@@ -399,6 +405,50 @@ func init() {
 	)
 	arguments.SetQuestion(fs, "wif-config", "WIF configuration:")
 	Cmd.RegisterFlagCompletionFunc("wif-config", arguments.MakeCompleteFunc(getWifConfigNameOptions))
+
+	addGcpEncryptionFlags(fs, &args.gcpEncryption)
+
+}
+
+func addGcpEncryptionFlags(fs *pflag.FlagSet, encryptionArgs *c.GcpEncryption) {
+
+	fs.StringVar(
+		&encryptionArgs.KmsKeyLocation,
+		KmsKeyLocationFlag,
+		"",
+		"The location of KMS keyring in GCP for custom encryption."+
```

Comment:
```suggestion
		"The location of KMS keyring in GCP for custom encryption. "+
```
Added a space after the period, otherwise the two senetences are together, as can be seen here:
```
      --kms-key-location string                               The location of KMS keyring in GCP for custom encryption.This should match the data center where the cluster's compute pool will be located.
      --kms-key-name string                                   The name of the KMS key in GCP to use for custom encryption.The key should belong to the KMS key ring specified.
      --kms-key-ring string                                   The name of the KMS key ring in GCP to use for custom encryption.The key ring should belong to the KMS location specified.
      --kms-key-service-account string                        The name of the service account in GCP with access to KMS keyring and key specified for custom encryption
```